IPython/terminal/ptutils.py

```diff
@@ -36,12 +39,16 @@ def _elide(string, *, min_elide=30):
     if len(string) < min_elide:
         return string
 
-    parts = string.split('.')
+    object_parts = string.split('.')
+    file_parts = string.split('/')
 
-    if len(parts) <= 3:
-        return string
+    if len(object_parts) > 3:
+        return '{}.{}\N{HORIZONTAL ELLIPSIS}{}.{}'.format(parts[0], parts[1][0], parts[-2][-1], parts[-1])
```

Comment:
you've kept `parts` here, which does not exist any more. Did you mean to replace with `object_parts`?

Same bellow with `file_parts` instead.